runtime/src/chplsys.c

```diff
@@ -52,6 +52,9 @@
 #include <errno.h>
 #include <unistd.h>
 
+#include <signal.h>
```

Comment:
have a look at the version I just merged, which puts them in an ordering I can be happy about too.

The reason I did that is that 'signal' sorts between 'pthread' and 'sched'. But maybe you have some partial ordering in mind...